test/mocha/tests.js

```diff
@@ -0,0 +1,34 @@
+/*global describe, it, xit*/
+function assert(expr, msg) {
+    if (!expr) {
+        throw new Error(msg || "failed");
+    }
+}
+
+describe("group a", function () {
+    it("foo", function () {
+        assert(5 === 5);
+        assert(1 / 0 === Infinity);
+    });
+
+    it("bar goes wrong", function () {
+        assert(5 === "5");
+    });
+});
+
+describe("group b", function () {
+    it("baz", function () {
+        assert(true);
+    });
+    xit("skipped test", function () {
```

Comment:
Is there also `xdescribe`? Should check for that. Not quite sure what to make of this: https://github.com/mochajs/mocha/commit/6cdca19eafa34bc8f05cfa69a8084fdd6b2cf06f